ola.spec.in

```diff
@@ -16,14 +15,13 @@ URL:		https://github.com/OpenLightingProject/ola
 Source:		https://github.com/OpenLightingProject/ola/releases/download/%{version}/%{name}-%{version}.tar.gz
 
 BuildRequires:	pkgconfig,pkgconfig(cppunit),pkgconfig(uuid),pkgconfig(avahi-client)
-BuildRequires:	pkgconfig(libusb-1.0) >= 1.0.2,pkgconfig(libmicrohttpd) >= 0.4.0
+BuildRequires:	pkgconfig(libusb-1.0) >= 1.0.2,pkgconfig(libmicrohttpd) >= 0.4.0%{!?el6:,pkgconfig(systemd)}
 BuildRequires:	libtool,bison,flex,python2-devel,openslp-devel,pkgconfig(libftdi%{!?el6:1}) >= 0.18
-BuildRequires:	protobuf-devel >= 0.2,protobuf-compiler,protobuf-python
+BuildRequires:	protobuf-devel >= 0.2,protobuf-compiler,protobuf-python,numpy
```

Comment:
I would rather let it be a buildrequire, in case we eventually do use them using ``` make all ``` or ``` make check ```